### DISCUSSION TITLE
Version 8.1 vs 7.2

### DISCUSSION THREAD

**Discussion by drmax24** [rebing/graphql-laravel#887] [Q&A]
Is version 8,1 somehow faster than 7.2? :p

**Comment by mfn** [comment]
TBH, I have not conducted proper benchmarks so I can't say.

The focus with the initial 8.0 release was getting rid of technical debt mostly and make the code base easier to maintain/understand and extensible in certain parts.

Running it in production I've not seen any performance issues. I did some basic Octane testing and here the performance is through the roof but that's unrelated to the changes itself (except: making it Octane compatible).
